### PR TITLE
Master remake form field

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.ts [hunk 1]
```diff
@@ -8,13 +8,13 @@ import {
   DxLoadPanelModule,
   DxSelectBoxModule,
   DxTextBoxModule,
-  DxToolbarModule,
+  DxToolbarModule, DxValidatorModule,
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.ts:11]: Move to a new line

File: packages/angular/src/styles.scss [hunk 2]
```diff
@@ -21,22 +21,52 @@
...
+      .dx-texteditor.form-editor .form-editor-input {
+        color: $accent-color;
+      }
+    }
+
+    .dx-texteditor,
+    .dx-texteditor.form-editor, {
+      pointer-events: none;
+
+      .form-editor-input {
```
Comment (dxvladislavvolkov) [inline packages/angular/src/styles.scss:55]: Could we move editor styles to form-editor.component?

**Review by dxvladislavvolkov — COMMENTED** [review]
Some comments

File: packages/angular/src/styles.scss [hunk 3]
```diff
@@ -21,22 +21,45 @@
...
 
-    dx-select-box task-priority {
+    task-priority {
       .separator {
         max-height: 12px;
       }
     }
   }
+
+  &.view-mode {
```
Comment (MikeVitik) [inline packages/angular/src/styles.scss:37]: Move this piece of styles to the form-editor component

**Review by MikeVitik — CHANGES_REQUESTED** [review]

**Review by dxvladislavvolkov — APPROVED** [review]

**Review by MikeVitik — DISMISSED** [review]

**Review by dxvladislavvolkov — DISMISSED** [review]

File: packages/angular/src/app/shared/components/form-datebox/form-datebox.component.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import {
+  Component, EventEmitter, Input, NgModule, Output, ViewChild,
+} from '@angular/core';
+import {
+  DxCalendarModule,
+  DxDropDownButtonModule,
+  DxDropDownButtonComponent, DxDateBoxModule,
```
Comment (groshenkovamarina) [inline packages/angular/src/app/shared/components/form-datebox/form-datebox.component.ts:7]: Move to a new line too

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**